framework/src/headers/mac.rs

```diff
@@ -90,7 +90,7 @@ impl Hash for MacAddress {
 pub struct MacHeader {
     pub dst: MacAddress,
     pub src: MacAddress,
-    etype: u16,
+    pub etype: u16,
```

Comment:
I don't think we should change this because we then allow the caller to ignore endianness.